holmes/plugins/toolsets/http/http_toolset.py

```
@@ -0,0 +1,425 @@
+                    url,
+                    headers=headers,
+                    auth=basic_auth,
+                    data=body,
+                    timeout=self._toolset.http_config.timeout_seconds,
+                    verify=self._toolset.http_config.verify_ssl,
+                )
+
+            # Return raw response (status + body)
+            try:
+                data = response.json()
+            except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)